src/reify.ml

```diff
@@ -1,3 +1,11 @@
+(**************************************************************************)
+(*  This is part of ATBR, it is distributed under the terms of the        *)
+(*         GNU Lesser General Public License version 3                    *)
```

Comment:
If this is really "or later", the headers should (in principle) contain this mention too.

Reply:
The "or later" comes from the `LICENSE` file, but is not reflected in the headers. Rather than having inconsistent headers, I thought it was better to just copy the header text from another file. We can update all headers at once at some point.